commit 88c4794f667d4f8272691c11f61ab3d033bae0cc (refs/changes/62/4762/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-08-29T10:43:53-07:00 (1 year, 1 month ago)
    
    MANTA-3125 deliver libmanta via npm registry instead of git repo (npmignore file to limit cruft from published package)
    Reviewed by: Jared Morrow <jm@joyent.com>
    Approved by: Jared Morrow <jm@joyent.com>

diff --git a/.npmignore b/.npmignore
new file mode 100644
index 0000000..3136e11
--- /dev/null
+++ b/.npmignore
@@ -0,0 +1,10 @@
+/.gitmodules
+/.dir-locals.el
+/Makefile
+/deps
+/docs
+/test
+/tools
+/tmp
+/npm-debug.log
+/libmanta-*.tgz
